src/hdx_cli_toolkit/cli.py

```diff
@@ -106,14 +108,24 @@ def decorator(f: FC) -> FC:
     default=None,
     help="A file path to export data from list to CSV",
 )
+@click.option(
+    "--with_extras",

```

Comment:
You should be able to set this to use a hyphen instead ("with-extras") but it will still turn into a variable with an underscore

Reply:
Automating the with_extras thing means I don't need to consider this. I think on the whole I prefer to have a consistent name rather than a magic conversion to a legal Python name